harvester/DBInterface.py

```diff
@@ -401,6 +401,8 @@ def write_record(self, record, repo_id, metadata_prefix, domain_metadata):
                     source_url = record["dc:source"][0]
                 else:
                     source_url = record["dc:source"]
+            elif 'source_url' in record:
```

Comment:
I think I forgot to remove lines 404-405 when I pulled the other source_url stuff -- I can get rid of this tomorrow.